modules/nw-multus-ipam-object.adoc

```diff
@@ -240,3 +240,52 @@ Whereabouts can be used for both IPv4 and IPv6 addresses.
 ifdef::sr-iov[]
 :!sr-iov:
 endif::[]
+
+[id="nw-multus-creating-whereabouts-reconciler-daemonset_{context}"]
+== Creating a Whereabouts-reconciler daemonset 
+
+The Whereabouts-reconciler is responsible for managing dynamic IP address assignments for the pods within a cluster using the Whereabouts IP Address Management (IPAM) solution. It ensures that each pods gets a unique IP address from the specified IP address range. It also handles IP address releases when pods are deleted or scaled down.
+
+[NOTE]
+====
+You can also use a `NetworkAttachmentDefinition` custom resource for dynamic IP address assignment. 
+====
+
+The Whereabouts-reconciler daemonset is automatically created when you configure an additional network through the Cluster Network Operator. It is not automatically created when you configure an additional network from a YAML manifest. 
+
+To trigger the deployment of the Whereabouts-reconciler daemonset, you must manually create a `whereabouts-shim` network attachment by editing the Cluster Network Operator `config.yaml` file.
+
+Use the following procedure to deploy the Whereabouts-reconciler daemonset. 
+
+.Procedure
+
+. Update your Cluster Network Operator `config.yaml` file to add the `whereabouts-shim` network attachment definition. For example:
```

Comment:
There's actually no `config.yaml` for this; The CNO configuration is what's considered a _singleton_, the object is created as part of cluster installation. Well, someone can modify the defaults for it, but that won't help someone here.

To do this, you'd need to use `oc edit` on the custom resource. I think this is what we're looking for:

`oc edit network.operator.openshift.io cluster`

And the `additionalNetworks` array will be in there, possibly populated with objects, if the user has already added some this way.

This approach is basically what's in here: `modules/nw-kube-proxy-configuring.adoc`